interactions/api/models/channel.py

```diff
@@ -322,6 +322,15 @@ class Tags(ClientSerializerMixin):  # helpers, hehe :D
 
     # Maybe on post_attrs_init replace emoji object with one from cache for name population?
 
+    @property
```

Comment:
Right now, the ID field of the Tags object is not converted to Snowflake but kept as an integer instead. Could you please change that too?

Reply:
pog